flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/MFD/A320_Neo_MFD.js

```diff
@@ -834,23 +834,32 @@ class A320_Neo_MFD_NDInfo extends NavSystemElement {
         }
         const ADIRSState = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum");
         const groundSpeed = Math.round(Simplane.getGroundSpeed());
+        const trueAirSpeed = SimVar.GetSimVarValue("AIRSPEED TRUE", "Knots");
+        const windDirection = SimVar.GetSimVarValue("AMBIENT WIND DIRECTION", "Degrees");
+        const windStrength = SimVar.GetSimVarValue("AMBIENT WIND VELOCITY", "Knots");
         const gs = this.ndInfo.querySelector("#GS_Value");
         const tas = this.ndInfo.querySelector("#TAS_Value");
         const wd = this.ndInfo.querySelector("#Wind_Direction");
         const ws = this.ndInfo.querySelector("#Wind_Strength");
         const wa = this.ndInfo.querySelector("#Wind_Arrow");
         const wptg = this.ndInfo.querySelector("#Waypoint_Group");
         const wptPPOS = this.ndInfo.querySelector("#Waypoint_PPOS");
-        if (ADIRSState != 2 || groundSpeed <= 100) {
-            //Hide TAS, and wind info
+        if (ADIRSState !== 2 || trueAirSpeed < 60) {
+            //TAS info Conditions
             tas.textContent = "---";
+        }
+        if (ADIRSState !== 2 || trueAirSpeed < 100 || windStrength < 2) {
+            //Wind Arrow info Conditions
             wd.textContent = "---";
             ws.textContent = "---";
             wa.setAttribute("visibility", "hidden");
         } else {
+            wd.textContent = (Math.round(windDirection)).toString().padStart(3, '0');
```

Comment:
Please check if the value actually changed

Reply:
Changed in what means?

Reply:
> 
> 
> Changed in what means?
> 
> Edit: the old code used a toFixed(0) so this should result in the same

Check if windDirection changed since last time textContent was updated. Changing textContent (or any DOM elements) unnecessarily is hard on performance.

Reply:
Ah ok
I'll try to do that then somehow

Reply:
> Please check if the value actually changed 

@Benjozork finally got some time to fix it. Please take a look :)